docs/docs/guides/server-deployment.md

```diff
@@ -204,19 +204,22 @@ To store logs using GCP Logging, set the `DSTACK_SERVER_GCP_LOGGING_PROJECT` env
 
 ## Metrics
```

Comment:
If there is a separate Metrics guide, this section can be removed from Server deployment.